Enhance cli release doc (#653)
kyma-project/community: docs/guidelines/releases-guidelines/03-kyma-cli-release-process.md

       ```bash
       brew bump-formula-pr --strict kyma-cli --url https://github.com/kyma-project/cli/archive/{RELEASE_VERSION}.tar.gz
       ```
      - With above command, it will create a PR to [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core) and contains version update changes to kyma-cli formula. Make sure all checks in this PR are passed, then the rest will be handled by Homebrew maintainers.

>       - If some tests failed, identify the root cause first, if it's not due to bugs, simply close this PR and re-run the above `brew bump-formula-pr` command with `--force` mode.

Reviewer: ```suggestion
      If some tests fail, identify the root cause.
      If it's not due to bugs, simply close your PR and re-run the previous `brew bump-formula-pr` command with `--force` mode.
```

>         - In case you got `fatal: a branch named 'bump-kyma-cli-{RELEASE_VERSION}' already exists`, please delete this local branch.

Reviewer: ```suggestion
        - If you got the message `fatal: a branch named 'bump-kyma-cli-{RELEASE_VERSION}' already exists`, delete this local branch.
```

>         - In case you got `Error: You need to bump this formula manually since the new URL and old URL are both: https://github.com/kyma-project/cli/archive/{RELEASE_VERSION}.tar.gz`, please restore the change in you local repo.

Reviewer: ```suggestion
        - If you got the message `Error: You need to bump this formula manually since the new URL and old URL are both: https://github.com/kyma-project/cli/archive/{RELEASE_VERSION}.tar.gz`, restore the change in you local repo. The local repo is under `/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core`.
```

I think the path to the repo belongs to this instruction right?

Author: Actually not only belongs to this instruction, it's also related to the first _In case you got `fatal: a branch named 'bump-kyma-cli-{RELEASE_VERSION}' already exists`, please delete this local branch._ Because you need to navigate to the local repo, then delete the branch, that's why I think probably better to put it in a dedicate bullet point. But if you think that's not needed, we can put them together.

Btw, I enhanced this sentence a bit to include different path for apple silicon mac, please review it again.

Reviewer: I see, thanks! 
Because the repo path is not a third option (with the error msgs being the first two options), you can remove the bullet and adjust indentation for that so that it's aligned with line 65 etc. 

Furthermore, get rid of the "please", no need for that. If we used "please" in instructions, there'd have to be a "please" in ever single line :)

>         - The local repo is under `/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core`.

Reviewer: ```suggestion
```

Author: OK, now you can have a final review @NHingerl
